app/routes/contact/components/ContactForm.tsx

```diff
@@ -54,7 +54,8 @@ const ContactForm = () => {
     () =>
       Promise.all([
         dispatch(fetchAllWithType(GroupType.Committee)),
-        dispatch(fetchGroup(REVUE_BOARD_GROUP_ID)),
+        // The revue board group does not exist in local dev environment
```

Comment:
Wouldn't it be better to add the fixture to have the revue board group? And if you do not intend to do so in this PR, could you change/add the comment to something along the lines of;
```
// TODO Remove `, { propagateError: false } ` after adding the revue board group to the fixtures
```

Reply:
Sure :)